lineman/app/services/mention_service.coffee

```diff
@@ -0,0 +1,7 @@
+angular.module('loomioApp').factory 'MentionService', ->
```

Comment:
I think I'm down with where you're going, particularly on not making a relation in the db. 

I'm happy to go with this, but still wish the server did not have to do a text transform, and that this transforming could be entirely client side. But.. I guess we can always change it later :)